src/main/java/javaslang/jackson/datatype/deserialize/BaseDeserializer.java

```diff
@@ -45,6 +44,13 @@ Object _deserialize(JsonParser jp, JavaType expectedType, DeserializationContext
 
     private Object _deserializeObject(JsonParser jp, JavaType expectedType, DeserializationContext ctx)
             throws IOException, ClassNotFoundException {
+        // TODO (hotfix)
```

Comment:
Javaslang objects can hold other javaslang objects, java maps <strong>and bean objects</strong> which should be deserialized by <code>RootValueDeserializer</code>